src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,21 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+interface CommentProps {
+  comment: Comment
+}
+export const CommentInfo: React.FC<CommentProps> = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{comment.name}</strong>
+
+      {' by '}
+
+      <a className="CommentInfo__email" href={`mailto:${comment.email}`}>
```

Comment:
you can destructure comment 
```suggestion
      <a className="CommentInfo__email" href={`mailto:${email}`}>
```